README.md

```diff
@@ -32,14 +32,16 @@ interface Contact {
 **NB:**, fields with a question mark (e.g. `county?`) indicate that the value may possibly be empty, i.e. no such key exists.  If no values are provided by the state, this is how it is indicated.
 
 ## Adding a New State
-Each state's crawler is put under it's own folder (e.g. `states/new_york/main.py`) with potentially other files in the folder.
-- We use `cache_request` to reqeust webpages so that the results are saved to a local cache for faster development.
-- You can invoke the file using PyInvoke, i.e. 
+Each state's crawler is put under its own folder (e.g. `states/new_york/main.py`), with potentially other files in the folder.
+- The goal is for each state's crawler to fetch and process all of its required inputs without human intervention, so that we can easily re-run scripts periodically to collect fresh data.
+- We use `cache_request` from `common.py` to request webpages so that the results are saved to a local cache for faster development. The `common` module also contains several other functions which may be useful, including `cache_selenium` and `cache_webkit`.
+- We generally avoid using intermediate data files, but if required these are saved in the same common cache folder for ease of clearing for fresh runs.
```

Comment:
Do we currently do this?  I thought the whole point of `cache_request` was to eliminate this.

Reply:
New Hampshire is the only exception -- it uses `scrapy` instead of `cache_request`. It has its own code to check if the scrapy output file exists before requerying.

Reply:
See pr #52 to use `cache_request` instead of scrapy. Let's remove this line from the readme.